app/endpoint/create-room/ValidateCreateRoomRequest.ts

```diff
@@ -0,0 +1,50 @@
+import {validate} from '../../lib/validatejs/0_13_1/validate';
+
+export default function validateRequest(query: any): any {
+
+    console.log('query: ', query);
+
+    const constraints = {
+        'visitor': {
+            presence: {
+                allowEmpty: false,
+            },
+        },
+        'visitor.department': {
+            presence: {
+                allowEmpty: false,
+            },
+            type: 'string',
+        },
+        'visitor.token': {
```

Comment:
Rename `token` to `contactUUid` because token can be anything and we need a `contactUuid` for start flows to it later.

Reply:
If that lib allows regex, you can use the following:

`[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}`

(check the characters escaping)

Reply:
Remember to take `contactUuid' param and set as `token` on visitor.